### ISSUE TITLE
Use consistently cased keys, preferably camelCase

### ISSUE THREAD

**Issue by laggingreflex** [perarnborg/vuex-oidc#25]
Option keys are inconsistent:

```
  silent_redirect_uri: ... <- snake_case
  automaticSilentRenew: ... <- camelCase
```
I'm trying to read these from environment variables where the standard case is generally CAPITAL_SNAKE, but usually there are libs to normalize them (into camelCase) but due to different cases, each key requires special attention.

**Comment by perarnborg** [comment]
@laggingreflex good point! The diffs in the casing are an artifact from oidc-client, that the library dependends on.

But I like the idea of normalizing the options. I will probably have this fixed within short.

**Comment by perarnborg** [comment]
@laggingreflex Fixed in v1.14.0
